app/src/main/java/com/picone/go4lunch/presentation/ui/MapsFragment.java

```diff
@@ -23,12 +50,85 @@ public static MapsFragment newInstance() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        mFusedLocationProviderClient = LocationServices.getFusedLocationProviderClient(requireActivity());
```

Comment:
Here are the tasks to performs at Fragment init : 

- Launch a loader : see [https://airbnb.design/introducing-lottie/](url)
- Check permissions are granted. Break here if permissions not granted
- Get User location
- Get The map
- Display User position on the map
- Release the loader